src/entityCreation/README.md

```diff
@@ -0,0 +1,141 @@
+# 🧱 Entity Management - Creation Flow
```

Comment:
@MallanagoudaB lets call the directory entityTypeAndEntityOnboarding instead entityCreation. inside that create another folder utilityScripts then move the scripts in that. modify the readme accordingly if required